src/wdioServiceAdapter.ts

```diff
@@ -0,0 +1,15 @@
+export function wdioService(serviceDefinition: string | [string, object?, object?, object?]): Service {
+    let servicePath = serviceDefinition;
```

Comment:
why `let` if it's not changing?

Reply:
there was destructuring in line 7, in case if module has configuration. But I refactored solution without let